extensions/host/internal.m

```diff
@@ -692,6 +690,7 @@ static int hs_vramSize(lua_State *L) {
                         if (ValueInBytes) Size >>= 20;
 
                         lua_pushnumber(L, Size) ;
+                        CFRelease(VRAMSize);
```

Comment:
Per https://developer.apple.com/documentation/iokit/1514537-ioregistryentrysearchcfproperty?language=objc - we are responsible for freeing the result of `IORegistryEntrySearchCFProperty()`

Reply:
Likely also a copy-paste mistake.